README.rdoc

```diff
@@ -282,4 +290,4 @@ When sorting links, the following sort orders are available.
 * <tt>:page_athority</tt>
 * <tt>:domain_authority</tt>
 * <tt>:domains_linking_page</tt>
-* <tt>:domains_linking_domain</tt>
\ No newline at end of file
+* <tt>:domains_linking_domain</tt>
```

Comment:
It's unclear to me what has changed on this line. I'm assuming it's whitespace of some variety.